src/HttpHelper/HttpClientHelper/HttpClient.php

```diff
@@ -103,7 +128,9 @@ private function request(
                 $iHttpCode = $e->getCode();
             }
 
-            Logger::error("$sAction : error while requesting $sMethod $sEffectiveUri - $iHttpCode", $logContext, \array_merge($aLogData, [
+            $sLogLevel = $bErrorLogLevelForHttpRequest ? 'error' : 'notice';
+
+            Logger::$sLogLevel("$sAction : error while requesting $sMethod $sEffectiveUri - $iHttpCode", $logContext, \array_merge($aLogData, [
```

Comment:
need to disable logs not change the level, we don't want to have `notice` for all error responses but disable default logs and handle them manually in the client, depending on the request specificity

Reply:
@leadinpier @pbernad-leadin 
Guys. Whose comment is more relevant?
https://lead-in.monday.com/boards/72721864/pulses/5154589834/posts/3560280413
or
https://lead-in.monday.com/boards/72721864/pulses/5154589834/posts/3566628920

Reply:
> @leadinpier @pbernad-leadin Guys. Whose comment is more relevant? https://lead-in.monday.com/boards/72721864/pulses/5154589834/posts/3560280413 or https://lead-in.monday.com/boards/72721864/pulses/5154589834/posts/3566628920

@rtwent my comment belonged to 
https://lead-in.monday.com/boards/72721864/pulses/4680529671
and not to
https://lead-in.monday.com/boards/72721864/pulses/5154589834

Reply:
Done